# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======

CounterDemoActionGoal action_goal
CounterDemoActionResult action_result
CounterDemoActionFeedback action_feedback
